src/components/search-query.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import PropTypes from 'prop-types';
+
+import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination } from 'react-instantsearch-dom';
+import webComponent from '../web-component';
+
+const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
+
+const hitComponent = ({ hit }) => (
+  <div>
+    { hit.last_name ?
+      <div><strong>Contributor:</strong> {hit.first_name} {hit.last_name}</div> : ''
+    }
+    <div><strong>Election: </strong>
+      <a href={`/election/${hit.election_location.toLowerCase()}/${hit.election_date}`} >
+        {hit.election_title}
+      </a>
+    </div>
+    { hit.title ?
+      <div><strong>Ballot Measure: </strong>
+        <a href={`/referendum/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.slug}`} >
+          {hit.title}
+        </a>
+      </div> : ''}
+    { hit.name ?
+      <div><strong>Candidate: </strong>
+        <a href={`/candidate/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.candidate_slug}`} >
+          {hit.name}
+        </a>
+      </div>
+     : ''
+    }
+    { hit.office_title ?
+      <div>
+        <strong>Office: </strong>
+        <a href={`/office/${hit.election_location.toLowerCase()}/${hit.election_date}/${hit.office_slug}`} >
+          {hit.office_title}
+        </a>
+      </div> : ''
+    }
+    { hit.amount ? <div><strong>Amount:</strong> ${hit.amount}</div> : ''}
+    <strong>Election date:</strong> {hit.election_date}<br />
+    <strong>Location:</strong> {hit.election_location}<br />
+  </div>
+);
+hitComponent.propTypes = {
```

Comment:
See comment above about React naming conventions.
```suggestion

HitComponent.propTypes = {
```